lib/workers/repository/update/branch/index.ts

```diff
@@ -209,9 +209,14 @@ export async function processBranch(
         };
       }
     }
+
+    logger.debug(
+      `Open PR Count: ${counts.get('PullRequests')?.current}, Existing Branch Count: ${counts.get('Branches')?.current}, Hourly PR Count: ${counts.get('HourlyPullRequests')?.current}`,
+    );
+
```

Comment:
Is this logged once per branch, regardless of whether any limits are set?

Reply:
Yes